src/screens/SearchScreen.tsx

```diff
@@ -46,6 +48,7 @@ const SearchScreen: FunctionComponent<SearchScreenProps> = ( { navigation } ) =>
           setInitPageCount={setPageCount}
         />
       </View>
+      <NavigationDrawer />
```

Comment:
This should go inside a `<Footer>` component (native-base)

Reply:
If this means that we need to create our own `BottomSheet` that uses native-base, then we may have to do that, otherwise if we mix and match techniques, we will end up with weird outcomes

Reply:
> This should go inside a `<Footer>` component (native-base)

So if I use `<Footer>` has some `native-base` pre defined area parameters (I guess) those would work if we don't want a floating drawer. 
<img width="1680" alt="image" src="https://user-images.githubusercontent.com/44710980/79776730-c6af4980-82fb-11ea-96fc-c91f9e83d377.png">

We just need to decide ratio we want our `<NavigationDrawer />`to have on the screen.
So for now the `BottomSheet` is using what I think is close to the mockup. Once we have a ratio we agree on, then it should render perfectly on all different sizes.